packages/devextreme/js/__internal/grids/grid_core/validating/m_validating.ts

```diff
@@ -1326,12 +1338,38 @@ export const validatingModule = {
 
               if (errorMessages.length) {
                 this._showValidationMessage($focus, errorMessages, column.alignment || 'left');
+                validationDescriptionValues.push(INVALID_MESSAGE_ID);
               }
             }
 
+            this._updateAriaValidationAttributes($focus, validationDescriptionValues);
             !isHideBorder && this._rowsView.element() && this._rowsView.updateFreeSpaceRowHeight();
           },
 
+          _updateAriaValidationAttributes($focus, inputDescriptionValues) {
+            if (inputDescriptionValues.length === 0) { return; }
+
+            const editMode = this._editingController.getEditMode();
+            const shouldSetValidationAriaAttributes = [
+              EDIT_MODE_CELL,
+              EDIT_MODE_BATCH,
+              EDIT_MODE_ROW].includes(editMode);
+
+            if (shouldSetValidationAriaAttributes) {
+              const $focusElement = this._getCurrentFocusElement($focus);
+              $focusElement.attr('aria-labelledby', inputDescriptionValues.join(' '));
+              $focusElement.attr('aria-invalid', true);
+            }
+          },
+
+          _getCurrentFocusElement($focus) {
+            let $focusElement = $focus;
```

Comment:
Why so difficult?

`_getCurrentFocusElement($focus) {
             if (this._editingController.isEditing()) {
                 return $focus.find(EDITORS_INPUT_SELECTOR).first();
             }
             return $focus;
 },`